src/base/bittorrent/session.cpp

```diff
@@ -375,6 +375,7 @@ Session::Session(QObject *parent)
     , m_announceToAllTiers(BITTORRENT_SESSION_KEY("AnnounceToAllTiers"), true)
     , m_asyncIOThreads(BITTORRENT_SESSION_KEY("AsyncIOThreadsCount"), 10)
     , m_hashingThreads(BITTORRENT_SESSION_KEY("HashingThreadsCount"), 2)
+	, m_maxActiveChecking(BITTORRENT_SESSION_KEY("MaxActiveChecking"), 1)
```

Comment:
Broken indentation is here and in several other places.

Reply:
Should be fixed now. I used tabs instead of spaces.